.devops/docker/start.sh

```diff
@@ -5,4 +5,5 @@ set -o pipefail
 set -o nounset
 set -o xtrace
 
+python ./manage.py migrate
```

Comment:
`migrate` should be run separately so that developers will have more extensive control of their local DB and it will be easier to react to any migrations errors that occur during deployment